home/import_assessments.py

```diff
@@ -360,6 +400,18 @@ class AssessmentRow:
     def fields(cls) -> list[str]:
         return [field.name for field in fields(cls)]
 
+    @classmethod
+    def check_missing_fields(cls, row: dict[str, str], row_num: int) -> None:
+        """
+        Checks for missing required fields in the row and raises an exception if any is missing.
+        """
+        missing_fields = [field for field in MANDATORY_HEADERS if field not in row]
+        if missing_fields:
+            raise ImportAssessmentException(
+                f"The import file is missing required fields: {', '.join(missing_fields)}",
+                row_num,
+            )
+
```

Comment:
Isn't this already covered by `AssessmentImporter.validate_headers`?

Reply:
No, validate_headers checks for missing headers before writing to rows. Check_missing_fields then checks that the mandatory headers do not have missing fields in its rows.

Reply:
All rows will have exactly the same set of keys, so if the headers aren't missing there they won't be missing here. If the intent is to check for missing _values_, we should change the code to do that instead of just checking the keys.

(Unless we're removing empty fields somewhere along the way and I missed it?)